consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java

```diff
@@ -345,7 +349,39 @@ private int overwriteLightblue(List<JsonNode> documentsToOverwrite) {
     protected Map<String, JsonNode> getDestinationDocuments(Map<String, JsonNode> sourceDocuments) {
         Map<String, JsonNode> destinationDocuments = new LinkedHashMap<>();
         if(sourceDocuments == null || sourceDocuments.isEmpty()){
-            LOGGER.info("Unable to fetch any destination documents as there are no source documents");
+            //LOGGER.info("Unable to fetch any destination documents as there are no source documents");
```

Comment:
Out of curiosity, in  case sourceDocuments to be null, would it be an IllegalArgumentException ? In summary I'm bit concerned if this could result into a inconsistent state or even if we forgot to log an problem (as I saw there was a log statement there )